docs/src/main/sphinx/connector/hive.rst

```diff
@@ -388,6 +388,12 @@ Property Name                                      Description
 ``hive.query-partition-filter-required``           Set to ``true`` to force a query to use a partition filter.   ``false``
                                                    You can use the ``query_partition_filter_required`` catalog
                                                    session property for temporary, catalog specific use.
+
+``hive.partition_statistics_sample_size``          You can use the ``hive.partition_statistics_sample_size``     100
```

Comment:
that sesssion property value is wrong .. should be all _

also explain property first